### PR TITLE
Version/search change for OS

### PR DESCRIPTION
**Pull request by frozenprocess**
## Description
Add: version dropdown to OSS Header
Fix: relocated searchbox into OSS Header
Delete: version warning removed from pages.

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
When I look at the deploy preview here: https://deploy-preview-4868--calico-master.netlify.app/maintenance/clis/calicoctl/configure/

I don't see the version dropdown working - it just shows a tiny empty box when I hover over it. Is that just because this is a preview so doesn't include all the versions?

**Comment by frozenprocess** [comment]
This happens because of two reasons

In a release, variable `site.archive` becomes true allowing the `/release` page to include the full list of previous releases.
https://github.com/projectcalico/calico/blame/1854fdb3d99f47ed786d5f6e92c1a35de7b55782/releases.md#L9

In order to keep the dropdown list up to date across current and previous releases it will make an AJAX call to the   `/releases` url and looks for a hidden `div` tag embeded in the page.
https://github.com/projectcalico/calico/pull/4868/files#diff-5dd90f94713cd356fad6b6cf0f293453aaf2dc06a2b01a803ef21d2fad3f0f7bR220


https://github.com/projectcalico/calico/pull/4913 <- Adds the hidden tag to the current latest version of the docs.

File: _config.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ nodecontainer: calico/node
 noderunning: calico-node
 
 open_source: true
+# Change this to v([0-9]+.[0-9])(?:.*?)$ for EE merge
```
Comment (caseydavenport) [inline _config.yml:27]: Why this comment? 

Also, why do we need to change it of the other docs site? Version scheme is more or less the same yeah?

Comment (frozenprocess) [reply _config.yml:27]: For EE docs website @bmckercher123 and @bcreane suggested a scheme like xx.y to be shown in the dropdown. In OSS this format leads to an incorrect visual representation of the version that is why I'm using a regex to seperate these two and EE pattern is different.

Comment (bcreane) [reply _config.yml:27]: @frozenprocess can you clean this up a bit to do the right thing for the calico/calico-private, e.g. https://github.com/projectcalico/calico/blob/master/_layouts/docwithnav.html#L183

**Comment by frozenprocess** [comment]
@caseydavenport I adjusted `version_pattern` as you suggested, both EE and OS will now show `minor.major` in the dropdown regardless of howmany digits it may contain.

I also changed the `releases.md` page logic a bit to preview the dropdown items. (You should be able to see the dropdown now in the preview.)

Let me know if there is anything else that can be improved

CC: @bcreane

File: css/search.scss [hunk 2]
```diff
@@ -1,8 +1,8 @@
 @import "../css/variables";
 
-.search-container {
-    display: none;
-}
+// .search-container {
```
Comment (bcreane) [inline css/search.scss:3]: if this is not needed and you don't anticipate bringing it back, please remove this block.

**Pull request merged**